addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/ImportFromAbstractDialog.java

```diff
@@ -150,6 +184,12 @@ protected void clear() {
         getTargetField().setText("");
     }
 
+    @Override
+    public void dispose() {
+        caller.getModel().getSession().removeOnContextsChangedListener(contextsChangedListener);
```

Comment:
This does not work correctly, the dialogue is disposed when done the import or it's closed.